### DISCUSSION TITLE
I cannot reach to my site via custom domain

### DISCUSSION THREAD

**Discussion by Atralbus** [vercel/community#1507] [Help] [answered]
### Summary

Hello,

I have an app  in vercel: [depremharita.vercel.app](https://depremharita.vercel.app/). It works fine in this domain but I wasn't able to get it working with a custom domain. I set up CloudFlare for the domain with the A record and CNAME given for the custom domain in Vercel. I updated the nameservers in my registrar. It's been 24 hours but I still can't reach to my site. When I check Whois for the domain I see that nameservers are updated but it's not working. Any idea what's wrong?

Custom domain: [depremharita.online](http://depremharita.online)
Error: ERR_NAME_NOT_RESOLVED (I guess which means it cannot find IP address for the given domain but I'm not sure if it's the only issue)

**Comment by swarnava** [comment]
Do you have Proxy enabled in Cloudflare? https://vercel.com/guides/using-cloudflare-with-vercel#with-proxy

**Comment by Atralbus** [comment, marked as answer]
It turned out the site was marked as `serverHold`. Now, it's working.
